Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/CancelDeletionProcessAsOwner/HandlerTests.cs

```diff
@@ -49,22 +49,20 @@ public void Cannot_start_when_given_identity_does_not_exist()
     {
         // Arrange
         var address = CreateRandomIdentityAddress();
-        var fakeIdentitiesRepository = A.Fake<IIdentitiesRepository>();
-
-        A.CallTo(() => fakeIdentitiesRepository.FindByAddress(address, CancellationToken.None, A<bool>._)).Returns<Identity?>(null);
-
-        var handler = CreateHandler(fakeIdentitiesRepository);
+        var handler = CreateHandler();
```

Comment:
The way it was before, I like better to be honest. There it was more explicit that it was set up to return null.

Reply:
Changes made.